clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/RateSensor.java

```diff
@@ -31,16 +37,26 @@ public double getRate() {
   /**
    * Record the rate value.
    */
-  public void record() {
-    sensor.record();
+  public void record(long now) {
+    rateAccumulator.increment();
```

Comment:
You convinced me that it's fine to use `LongAdder` here, but I find it weird to use both that and the Tehuti sensor... could you make use of the `LongAdderRateGauge` I'm adding in PR #300 and use just that, rather than both mechanisms?